### PR TITLE
Portfolio: details popup window

### PR DESCRIPTION
**Pull request by shm04**
## Popup window

In this pull request, I add the following: 

- Mobile version popup window.
- Desktop version popup window.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -51,4 +51,51 @@ function navHighlighter() {
...
+  <div class="buttons">
+  <button class="see-live" type="button">See live<img class="download" src="imgs/ic_link.png"></button>
+  <button class="see-src" type="button">See source<img class="download" src="imgs/github-black.png"></button>
+  </div>
+  <div class="arrow-btns">
+    <button class="previous" type="button"><img src="imgs/arrow-l.png">Previous project</button>
+    <button class="next" type="button">Next project<img src="imgs/arrow-r.png"></button>
+  </div>
+  </div>
+  `;
```
Comment (Qoosim) [inline script.js:92]: - [ ] It is required to create array of objects for all the  projects in your portfolio, loop through the array to display them them dynamically on page load. Your pop-window will also come from the array of objects you've created. This is contained in the project requirements. Kindly take a closer look and refactor your code.

project-rqt

**Review by Qoosim — COMMENTED** [review]
#  *Status: invalid Request* :red_circle: 

Hi @team, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] has commits from only one student. Every students in your pair-programming projects must have at least one commit. Kindly ensure your partner also contribute to this PR.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 2]
```diff
@@ -29,7 +29,7 @@ <h1 class="title">Sebastián Hernández</h1>
         <p class="text">Hello! I am a software developer! I can help you build a product, feature or website. Take a look of my works.
             If you like what you see and have a project you need coded, don’t hesitate and contact me.
         </p>
-        <button class="start-btn" href="#">Start collaboration</button>
+        <button class="start-btn" type="button" href="#">Start collaboration</button>
     </section>
     <section id="portfolio" class="projects">
         <div class="container-one">
```
Comment (Ibizugbe) [inline index.html:35]: Nice work building the project section but don't forget😉, It is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md) that the project section be generated dynamically from the array of objects created in your javascript code😉 kindly make the necessary changes🤗

pop__instruction

File: script.js [hunk 3]
```diff
@@ -51,4 +51,614 @@ function navHighlighter() {
...
+  <div class="buttons">
+  <a class="see-live" href=${projects.projectOne.live}>See live<img class="download" src="imgs/ic_link.png"></a>
+  <a class="see-src" href=${projects.projectOne.source}>See source<img class="download" src="imgs/github-black.png"></a>
+  </div>
+  <div class="arrow-btns">
+    <button class="previous" type="button"><img src="imgs/arrow-l.png">Previous project</button>
+    <button class="next" type="button">Next project<img src="imgs/arrow-r.png"></button>
+  </div>
+  </div>
+  `;
```
Comment (Ibizugbe) [inline script.js:150]: Nice job adding the functionalities of the popup👍 but currently, I think your popup is missing some features😉😉

- [ ]  When the close button is clicked, the popup should disappear, this is currently missing in your project😉😉
- [ ] The popup should stay fixed to the page when it is opened, currently, yours scrolls along with the page😉
- [ ] The pop should stick to the design both on mobile and desktop screens 😉 currently we can make some improvements on your desktop popup by ensuring the buttons are positioned center as given in the design🤗 kindly take note of the image below😉

design
popuo_template555desktopdesign

yours
your_mobilepopuptemplate5

kindly take note of the difference in the images used,, and the spacings between the various contents and make the necessary change🤗

kindly note that this applies to the mobile version too. kindly go through the design for mobile and ensure you stick to the design😉

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @shm04 👋, 

68747470733a2f2f6d65646961342e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966


Good job so far!👍
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Github flow practice has been used👍
- [x] An array of object containing the data for the popup and project section has been created👍


## Required Changes ♻️

- [ ] Currently, ESliint hasn't been setup in your project😉 remember the eslint checks for errors or wrong practices in your javascript and it is a requirement that we set it up for this milestone😉 kindly add your eslint and set it up properly🤗
- [ ]  Kindly ensue to stick to the given design as I have pointed in the comments below🤗 this is a requirement in this milestone that our popup looks like the design🤗🤗🤗

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shm04** [comment]
Sorry, but I think the Eslint files have been installed since the other pull request, I don´t know why you are asking to install it.
Also, I think my popup is created dynamically, no popup codes are on HTML, if I´m wrong please tell me why.
Also my close buttons it's working as it supposed to, I don't know why the reviewer said that it isn't working.
The other changes are made.

Thankyou 😄

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @shm04   👋 

STATUS: APPROVED!!! 🎉 🎉 🎉 🍾 🍾 🍾


Good job !!!
To Highlight! 👏 🟢
✔️ Good commit messages
✔️ Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 🎉 🎉 🍾 🍾 🍾



Optional suggestions
Check the comments under the review.

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack using this handle @mihndim.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**